drivers/spi/spi_ll_stm32.c

```diff
@@ -858,37 +858,30 @@ static void spi_stm32_irq_config_func_##id(const struct device *dev)		\
 #define STM32_SPI_IRQ_HANDLER(id)
 #endif
 
-#define DMA_CHANNEL_CONFIG(id, dir)					\
-		DT_INST_DMAS_CELL_BY_NAME(id, dir, channel_config)
-#define DMA_FEATURES(id, dir)						\
-		DT_INST_DMAS_CELL_BY_NAME(id, dir, features)
-#define DMA_CTLR(id, dir)						\
-		DT_INST_DMAS_CTLR_BY_NAME(id, dir)
-
 #define SPI_DMA_CHANNEL_INIT(index, dir, dir_cap, src_dev, dest_dev)	\
-	.dma_dev = DEVICE_DT_GET(DMA_CTLR(index, dir)),			\
```

Comment:
For consistency, either do the same change in uart driver in this commit as well.
Either move this to another commit

Reply:
in this commit ab764e3335214bfc8ef4a036b851d7f03e96521f
```
-	.dma_dev = DEVICE_DT_GET(DMA_CTLR(index, dir)),			\
+	.dma_dev = DEVICE_DT_GET(STM32_DMA_CTLR(index, dir)),			\
```
 for _both_ drivers drivers/spi/for spi_ll_stm32.c  and drivers/serial/uart_stm32.c